### DISCUSSION TITLE
Feature Request: displaying the distance to the next 3 resting places when traveling on a motorway / Autobahn

### DISCUSSION THREAD

**Discussion by wehkah** [osmandapp/OsmAnd#20568] [Ideas]
Hello,

I would like to include the distance to the next 2 or 3 resting places / motorway stations on either side of the screen when travelling on a motorway. I know that there are road signs announcing those places, but those signs are always sort on short notice. If you need to keep an eye of your driving time, like truck drivers need to do, it is important to know when and where a break can be made.

I know I can make resting places be displayed as POI, but these POI are displayed not before they fit on the screen.

**Comment by mikehgentry** [comment]
Similar to #18007
